src/fido/err.h

```diff
@@ -76,6 +76,7 @@
 #define FIDO_ERR_UV_BLOCKED		0x3c
 #define FIDO_ERR_UV_INVALID		0x3f
 #define FIDO_ERR_UNAUTHORIZED_PERM	0x40
+#define FIDO_ERR_USER_CANCELLED	0x41
```

Comment:
Would you be amenable to using FIDO_ERR_USER_CANCELLED 0xf0 ?

Reply:
In CTAP2, the canonical error for a cancelled operation is `CTAP2_ERR_OPERATION_DENIED`. The user is effectively denying the application access to the operation. The specification, concerning itself mainly with authenticator behavior, does not say how an application should react to this particular error. If it so chooses, it may track it as a failed attempt.

It's a similar situation in WebAuthn, the error would be mapped to a `NotAllowedError`, which may be returned for a number of reasons.